nifi-nar-bundles/nifi-gcp-bundle/nifi-gcp-processors/src/main/java/org/apache/nifi/processors/gcp/storage/PutGCSObject.java

```diff
@@ -291,6 +295,8 @@ public List<PropertyDescriptor> getSupportedPropertyDescriptors() {
         descriptors.add(BUCKET);
         descriptors.add(KEY);
         descriptors.add(CONTENT_TYPE);
+        descriptors.add(RESOURCE_TRANSFER_SOURCE);
+        descriptors.add(FILE_RESOURCE_SERVICE);
         descriptors.add(CRC32C);
         descriptors.add(ACL);
```

Comment:
I would move the new properties up, before `CONTENT_TYPE`. In that way, the properties would be "grouped" like:
- target (Bucket, Key)
- source
- metadata (Content Type, CRC, etc)
```suggestion
        descriptors.add(BUCKET);
        descriptors.add(KEY);
        descriptors.add(RESOURCE_TRANSFER_SOURCE);
        descriptors.add(FILE_RESOURCE_SERVICE);
        descriptors.add(CONTENT_TYPE);
        descriptors.add(CRC32C);
        descriptors.add(ACL);
```

Reply:
thanks, committed the suggestion.